libraries/wasm-jit/Source/Runtime/CMakeLists.txt

```diff
@@ -31,6 +31,8 @@ add_definitions(${LLVM_DEFINITIONS})
 
 add_definitions(-DRUNTIME_API=DLL_EXPORT)
 
+target_include_directories( Runtime PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/../../../chain/include )
```

Comment:
Gross. Any thoughts on improving this? chain library uses wasm-jit so I can't create some sort of cyclic dependency where wasm-jit uses chain, I don't think

Reply:
Can we reverse the direction?  It looks like we only include from this directory to use `eosio::chain::wasm_constraints::maximum_table_elements`

If we define that as a constant in say `Runtime.h` in wasm-jit, we can then include that in our chain level file and set our value to it.